src/endpoints/products/products.ts

```diff
@@ -50,6 +51,8 @@ export interface ProductsSearchEndpointParameters {
   pricePromotionKey?: string;
 
   minProductId?: number;
+
+  orFiltersOperator?: ArrayMinLength<string | number, 2>;
```

Comment:
why do we accept also `numbers` here?

the parameter actually only accepts attribute group keys and not IDs

Reply:
I see, generally, I'd recommend using the key since for certain filters this is also optimized and makes the API faster.